### PR TITLE
Experimental: building triangles + minor code fixes.

### PR DESCRIPTION
**Pull request by vng**

### CODE REVIEW

File: libs/routing/edge_estimator.cpp [hunk 1]
```diff
@@ -346,6 +346,8 @@ class BicycleEstimator final : public EdgeEstimator
 // CarEstimator ------------------------------------------------------------------------------------
 class CarEstimator final : public EdgeEstimator
 {
+  mutable double m_lastETASpeed = -1;
```
Comment (biodranik) [inline libs/routing/edge_estimator.cpp:350]: Fix unused variable warning:
```suggestion
#ifdef DEBUG
  mutable double m_lastETASpeed = -1;
#endif
```

File: libs/geometry/geometry_tests/robust_test.cpp [hunk 2]
```diff
@@ -29,6 +29,16 @@ bool InsideTriangle(P const & p, P const ps[])
   return IsPointInsideTriangle(p, ps[0], ps[1], ps[2]);
 }
 
+UNIT_TEST(OrientedS_CrossProduct)
+{
+  P arr[] = {{1, 2}, {3, 4}, {5, 6}};
```
Comment (biodranik) [inline libs/geometry/geometry_tests/robust_test.cpp:34]: Test for edge cases?

**Review by biodranik — APPROVED** [review]
